src/Resources/contao/dca/tl_google_map_overlay.php

```diff
@@ -355,6 +357,50 @@
             'eval'      => ['rgxp' => 'digit', 'maxlength' => 10, 'tl_class' => 'w50', 'mandatory' => true],
             'sql'       => "int(10) unsigned NOT NULL default '0'"
         ],
+        'kmlUrl'                 => [
+            'label'     => &$GLOBALS['TL_LANG']['tl_google_map_overlay']['kmlUrl'],
+            'exclude'   => true,
+            'search'    => true,
+            'inputType' => 'text',
+            'eval'      => ['rgxp' => 'url', 'decodeEntities' => true, 'maxlength' => 255, 'tl_class' => 'w50', 'mandatory' => true],
```

Comment:
Maybe we could even add ` 'dcaPicker'=>true,` to get the picker wizard - but I have to test if this urls are working!

Reply:
Agreed. Would be good if you could check that and add it.

Reply:
The URL to the kml needs to be absolute and with http! Otherwise google wont use it as overlay. That's why I wouldn't recommend implementing the picker yet. 
Nothing has changed in the last 10 years - I vaguely remember that I already had that as problem when the first version of my current relaunch was being built.

Reply:
Do you think, we should add this ("needs to be absolute and with http") to the label as a hint and in the readme?